### DISCUSSION TITLE
Enhancement: cerebras API Support

### DISCUSSION THREAD

**Discussion by Fusion-Parth** [danny-avila/LibreChat#5153] [Feature Requests & Suggestions]
### What features would you like to see added?

API Integration: Add support for the Cerebras.ai API, allowing users to input their API key and leverage its capabilities.

### More details

I'd like to request support for integrating the Cerebras.ai API into LibreChat. I have an API key and would like to use it with the platform. This feature would allow users to connect and utilize the Cerebras.ai services seamlessly.

### Which components are impacted by your request?

General

### Code of Conduct

- [X] I agree to follow this project's Code of Conduct

**Comment by danny-avila** [comment]
If it’s OpenAI compatible, it should already be supported: https://www.librechat.ai/docs/quick_start/custom_endpoints

**Reply by Dual-0** [reply]
look at my config example in #4806
